source/features/view-last-pr-deployment.tsx

```diff
@@ -14,9 +14,10 @@ function init(): void {
 	}
 
 	const {href} = select.last<HTMLAnchorElement>(deploymentSelector)!;
-	select('.gh-header-actions')!.prepend(
+	const headerActions = select('.gh-header-actions')!;
+	headerActions.prepend(
 		<a
-			className="rgh-last-deployment btn btn-sm d-none d-md-block"
+			className={`rgh-last-deployment btn btn-sm d-none d-md-block ${select.exists('[aria-label="Edit Pull Request title"]', headerActions) ? 'mr-1' : ''}`}
```

Comment:
I don't mind having a little extra space to save all of this code here.

This diff should just include an unconditional `mr-1`

Reply:
What do you think about https://github.com/sindresorhus/refined-github/issues/4277#issuecomment-826358707?

Perhaps add a 4px left margin to `Open With` in `refined-github.css` so it looks better and consistent.

---

Also I find that on the issue page the spacing is 8px:

![image](https://user-images.githubusercontent.com/44045911/116057907-da7d5080-a6b1-11eb-8e85-8cc2ee60db4b.png)

vs PR page:

![image](https://user-images.githubusercontent.com/44045911/116057929-e0733180-a6b1-11eb-9522-fd823f477b81.png)